tableauserverclient/server/request_options.py

```diff
@@ -62,24 +62,49 @@ def apply_query_params(self, url):
         return "{0}?{1}".format(url, '&'.join(params))
 
 
-class ImageRequestOptions(RequestOptionsBase):
+class _FilterOptionsBase(RequestOptionsBase):
+    def __init__(self):
+        self.view_filters = []
```

Comment:
Calling it view filters and then using it in CSV options is a little confusing to me -- can you tell me your thinking on this vs just 'filters'?

I'm not opposed just want to understand.

Reply:
We already have filters which is for filtering responses to the "get" calls.  View Filters are different because they are filtering the view.  A CSV is still a view of the data...

Reply:
These are filtering literal Views (Vizzes?) Ok that makes sense.